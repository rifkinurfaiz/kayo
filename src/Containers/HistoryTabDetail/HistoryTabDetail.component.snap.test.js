import * as React from 'react';
import { shallow } from 'enzyme';

import HistoryTabDetailComponent from './HistoryTabDetail.component';

jest.mock('react-native-swiper', () => 'Swiper');

describe('#HistoryTabDetailComponent', () => {
  it('should render correctly', () => {
    expect(shallow(<HistoryTabDetailComponent />)).toMatchSnapshot();
  });
});
