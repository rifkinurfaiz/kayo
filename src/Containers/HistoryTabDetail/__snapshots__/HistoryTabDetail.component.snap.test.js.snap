// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#HistoryTabDetailComponent should render correctly 1`] = `
<Swiper
  loop={false}
  showsPagination={false}
>
  <View>
    <HistoryTabDetailContent
      data={
        Array [
          Object {
            "assets": Array [
              Object {
                "amount": "Rp10.000.000",
                "id": "jenius",
                "name": "Jenius",
              },
              Object {
                "amount": "Rp10.200.140.234",
                "id": "bca",
                "name": "BCA",
              },
              Object {
                "amount": "Rp2.040.000",
                "id": "ovo",
                "name": "OVO",
              },
              Object {
                "amount": "Rp1.000",
                "id": "gopay",
                "name": "Gopay",
              },
            ],
            "month": "January",
            "totalAmount": "Rp212.181.234",
          },
          Object {
            "assets": Array [
              Object {
                "amount": "Rp10.000.000",
                "id": "jenius",
                "name": "Jenius",
              },
              Object {
                "amount": "Rp10.200.140.234",
                "id": "bca",
                "name": "BCA",
              },
              Object {
                "amount": "Rp2.040.000",
                "id": "ovo",
                "name": "OVO",
              },
              Object {
                "amount": "Rp1.000",
                "id": "gopay",
                "name": "Gopay",
              },
            ],
            "month": "February",
            "totalAmount": "Rp212.181.234",
          },
        ]
      }
      key="0"
      year={2020}
    />
  </View>
  <View>
    <HistoryTabDetailContent
      data={
        Array [
          Object {
            "assets": Array [
              Object {
                "amount": "Rp10.000.000",
                "id": "jenius",
                "name": "Jenius",
              },
              Object {
                "amount": "Rp10.200.140.234",
                "id": "bca",
                "name": "BCA",
              },
              Object {
                "amount": "Rp2.040.000",
                "id": "ovo",
                "name": "OVO",
              },
              Object {
                "amount": "Rp1.000",
                "id": "gopay",
                "name": "Gopay",
              },
            ],
            "month": "January",
            "totalAmount": "Rp212.181.234",
          },
          Object {
            "assets": Array [
              Object {
                "amount": "Rp10.000.000",
                "id": "jenius",
                "name": "Jenius",
              },
              Object {
                "amount": "Rp10.200.140.234",
                "id": "bca",
                "name": "BCA",
              },
              Object {
                "amount": "Rp2.040.000",
                "id": "ovo",
                "name": "OVO",
              },
              Object {
                "amount": "Rp1.000",
                "id": "gopay",
                "name": "Gopay",
              },
            ],
            "month": "February",
            "totalAmount": "Rp212.181.234",
          },
          Object {
            "assets": Array [
              Object {
                "amount": "Rp19.000.000",
                "id": "jenius",
                "name": "Jenius",
              },
              Object {
                "amount": "Rp10.200.140.234",
                "id": "bca",
                "name": "BCA",
              },
              Object {
                "amount": "Rp2.040.000",
                "id": "ovo",
                "name": "OVO",
              },
              Object {
                "amount": "Rp1.000",
                "id": "gopay",
                "name": "Gopay",
              },
            ],
            "month": "March",
            "totalAmount": "Rp221.181.234",
          },
          Object {
            "assets": Array [
              Object {
                "amount": "Rp19.000.000",
                "id": "jenius",
                "name": "Jenius",
              },
              Object {
                "amount": "Rp10.200.140.234",
                "id": "bca",
                "name": "BCA",
              },
              Object {
                "amount": "Rp2.040.000",
                "id": "ovo",
                "name": "OVO",
              },
              Object {
                "amount": "Rp1.000",
                "id": "gopay",
                "name": "Gopay",
              },
            ],
            "month": "April",
            "totalAmount": "Rp221.181.234",
          },
          Object {
            "assets": Array [
              Object {
                "amount": "Rp19.000.000",
                "id": "jenius",
                "name": "Jenius",
              },
              Object {
                "amount": "Rp10.200.140.234",
                "id": "bca",
                "name": "BCA",
              },
              Object {
                "amount": "Rp2.040.000",
                "id": "ovo",
                "name": "OVO",
              },
              Object {
                "amount": "Rp1.000",
                "id": "gopay",
                "name": "Gopay",
              },
            ],
            "month": "May",
            "totalAmount": "Rp221.181.234",
          },
        ]
      }
      key="1"
      year={2021}
    />
  </View>
</Swiper>
`;
